synapseutils/walk.py

```diff
@@ -11,6 +26,9 @@ def walk(syn, synId):
 
     :param synId:          A synapse ID of a folder or project
 
+    :param includeTypes:   Must be a list of entity types (ie. ["folder","file"]) which can be found here:
+                           http://docs.synapse.org/rest/org/sagebionetworks/repo/model/EntityType.html
+
     Example::
```

Comment:
Whoa:  Does this work, that would mean that the return tuple would be different depeneding on how you called this function.  That seems confusing from a user perspective and also incompatible with how os.walk works.

Is there a broader goal here?